### DISCUSSION TITLE
Spline UI not rendering on AWS ec2

### DISCUSSION THREAD

**Discussion by sridevijms** [AbsaOSS/spline-spark-agent#214] [Q&A]
Hi,

I have set up spline and spark on AWS ec2 instance. I am able to see consumer execution events as well. However, the lineage is not rendered on the UI. 

When I started spline using docker compose up , it is failing with the error "Error has been occurred during communication with a server".
 So, I started with the option DOCKER_HOST_EXTERNAL=152.46.108.145 docker-compose up and I am not seeing the error.  Please let me know if this the right way to start spline server and let me know how to get this UI rendered. 

Thanks
Sridevi.

**Comment by wajda** [comment]
Regarding `DOCKER_HOST_EXTERNAL`, if you see the execution events on the UI then you have it done correctly. `DOCKER_HOST_EXTERNAL` address must be visible from the client browser, not the server. This is the address the browser connects to to reach Spline REST gateway.

Regarding "the lineage is not rendered", can you give more details on that? Do you see an empty page? Are there any errors in the browser console or the server logs?
